[2024-09-22T11:28:01.550+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-22T11:28:01.896+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_setup_teardown_taskflow.my_second_task scheduled__2024-09-20T00:00:00+00:00 [queued]>
[2024-09-22T11:28:01.994+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_setup_teardown_taskflow.my_second_task scheduled__2024-09-20T00:00:00+00:00 [queued]>
[2024-09-22T11:28:02.004+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-09-22T11:28:02.527+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): my_second_task> on 2024-09-20 00:00:00+00:00
[2024-09-22T11:28:02.547+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=883) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-22T11:28:02.549+0000] {standard_task_runner.py:64} INFO - Started process 886 to run task
[2024-09-22T11:28:02.550+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'example_setup_teardown_taskflow', 'my_second_task', 'scheduled__2024-09-20T00:00:00+00:00', '--job-id', '2607', '--raw', '--subdir', '/home/***/.local/lib/python3.12/site-packages/***/example_dags/example_setup_teardown_taskflow.py', '--cfg-path', '/tmp/tmpi3z9dfuj']
[2024-09-22T11:28:02.553+0000] {standard_task_runner.py:91} INFO - Job 2607: Subtask my_second_task
[2024-09-22T11:28:02.958+0000] {task_command.py:426} INFO - Running <TaskInstance: example_setup_teardown_taskflow.my_second_task scheduled__2024-09-20T00:00:00+00:00 [running]> on host 1d2bd9c1668e
[2024-09-22T11:28:03.469+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_setup_teardown_taskflow' AIRFLOW_CTX_TASK_ID='my_second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-09-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-20T00:00:00+00:00'
[2024-09-22T11:28:03.471+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-22T11:28:03.472+0000] {logging_mixin.py:188} INFO - Hello 2
[2024-09-22T11:28:03.472+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-09-22T11:28:03.473+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-22T11:28:03.488+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=example_setup_teardown_taskflow, task_id=my_second_task, run_id=scheduled__2024-09-20T00:00:00+00:00, execution_date=20240920T000000, start_date=20240922T112801, end_date=20240922T112803
[2024-09-22T11:28:03.819+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-09-22T11:28:03.862+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-22T11:28:04.087+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
